personal-growth/becoming-a-mentor.md

```diff
@@ -0,0 +1,118 @@
+- **Create accountability.** It is extremely important for every one-on-one where problems have been
+  raised to create an action plan. Make sure both you and your mentee are accountable for that plan,
+  and monitor its progress closely.
+
+One more tip on scheduling the calls: when you have many mentees, it can be tempting to schedule all
+of the one-on-ones on Friday, so that the rest of your week is free to do other work. However, as
+these meetings are very demanding, it is hard to really focus on more than one or two in the same
+day. Instead, try to spread them out throughout the week, so that you can give the same level of
+attention to the same one-on-one.
+
+## Giving feedback
+
+- Don't wait until the 1:1, give feedback immediately
+- Praise in public, criticize in private
+- Make it about their growth, not their job/responsibilities
+- Listen to the response: sometimes people have good reasons for behaving the way they do
+  - If someone misses all standups for a project perhaps the standups are useless and should be
+    restructured?
+
+## Promoting growth
+
+- Identify areas where the mentee wants to grow
+  - These can be proposed by the mentee or identified by you on a project
+- Find projects the mentee can work on to grow in those areas
+  - These can be Friday projects or client-related projects
+  - Never compromise the client's project just to make a mentee grow (this can be avoided by pairing
+    the mentee with someone who has more experience, or ensuring in any way that the mentee's work
+    doesn't disrupt the client's plans)
+- Fundamentals to keep in mind: English, usage of version control, communication, priority 
+  management
+- Do your best to foster mentors
+
+## Managing projects
+
+- Be proactive: you should be the client's best advisor
+- Act as if you're spending your own money, not the client's
+- When you need to make a decision, listen to feedback and incorporate it in your thought process, 
+  but at the end of the day you need to decide, commit and execute
+  - Teach people to disagree and commit
+
+## Handling conflict
+
+- Don't take it personally: you have a responsibility of doing what's best for Nebulab
+- Take your time to reply. Even if you're in a meeting, it is perfectly acceptable to tell the other
+  person that you will collect your thoughts and then get back to them with an answer. This is much
+  better than giving the wrong answer because you're too nervous to think
+- You don't need to be apologetic if there's no reason, but also, you should only respond with 
+  facts and numbers, and always be empathetic towards the other person (gently show them that 
+  where they're wrong)
+- Always end with a summary of what has been discussed and a plan both parties commit to that will
+  make things right. Also establish future checkpoints to see how the situation is progressing
+- If things get out of hand or you see the situation is stalling, escalate in #growth
```

Comment:
Maybe we should specify what `#growth` is here? I think it will be read by non-mentors or people outside the organization as well. :)